app/app.go

```diff
@@ -797,7 +842,7 @@ func New(
 		slashingtypes.ModuleName,
 		evidencetypes.ModuleName,
 		stakingtypes.ModuleName,
-		ibchost.ModuleName,
```

Comment:
Pretty sure we need to remove upgradetypes.ModuleName from BeginBlockers and add it to PreBlockers. "When using (legacy) application wiring, the following must be added to app.go"

```
app.mm.SetOrderPreBlockers(
  upgradetypes.ModuleName,
)
```

Reply:
I've got it in the list of TODOs in the [epic](https://github.com/provenance-io/provenance/issues/1760).